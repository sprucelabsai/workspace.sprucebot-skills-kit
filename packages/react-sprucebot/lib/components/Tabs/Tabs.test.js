'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

require('jest-styled-components');

var _Tabs = require('./Tabs');

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

test('Tabs renders', function () {
	var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_Tabs.Tabs, null)).toJSON();
	expect(tree).toMatchSnapshot();
});

test('TabPane renders', function () {
	var tree = _reactTestRenderer2.default.create(_react2.default.createElement(_Tabs.TabPane, null)).toJSON();
	expect(tree).toMatchSnapshot();
});