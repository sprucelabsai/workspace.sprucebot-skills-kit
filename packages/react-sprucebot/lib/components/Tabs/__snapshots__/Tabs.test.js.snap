// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabPane renders 1`] = `
<div
  className="tap__pane"
/>
`;

exports[`Tabs renders 1`] = `
<div
  className="tabs__container "
>
  <div
    className="tabs"
  />
  <div
    className="tab__panes"
  />
</div>
`;
